lib/tokenize.js

```diff
@@ -313,6 +318,29 @@ module.exports = function tokenize (input, options) {
             next = regex.lastIndex - 2;
           }
 
+          // Exponential number notation with minus or plus: 1e-10, 1e+10
+          if (regex === wordEndNum || code === period) {
+            let code1 = css.charCodeAt(next),
```

Comment:
let's get some better descriptors for variable names here. `ncode` `ncode1` `ncode2` would be a little better since it'd correspond with the values being pulled. we don't want to do nextCode, nextCodeNext, nextCodeNextnext or anything silly like that, but we can do better than code1-3.